### PR TITLE
Bump versions of serialization dependencies

### PR DESCRIPTION
**Pull request by jonhoo**

### CODE REVIEW

**Comment by jonhoo** [comment]
@giltene Any chance you could enable travis builds for this repository so we get CI?

**Comment by jonhoo** [comment]
@marshallpierce All the tests pass locally for me, but would you mind also running the tests on your end just as a double-check given that we don't have CI at the moment?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
-- `crossbeam-channel` dependency was bumped to 0.5
 
 ### Changed
+- `crossbeam-channel` dependency was bumped to 0.5
+- `nom` dependency was bumped to 6.0.1
+- `base64` dependency was bumped to 0.13
```
Comment (marshallpierce) [inline CHANGELOG.md:12]: should probably mention a bump in minimum rust version here (which I'm 100% on board with fwiw)

**Review by marshallpierce — APPROVED** [review]
all tests passed for me 👍

**Comment by jonhoo** [comment]
Released as `7.2.0` :tada:

**Pull request merged**